controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -1262,3 +1263,104 @@ def test_identity_probe(self):
         request = Mock()
         context = Mock()
         self.servicer.Probe(request, context)
+
+
+class TestControllerServerValidateVolumeCapabilities(AbstractControllerTest):
+
+    def get_create_object_response_method(self):
+        return csi_pb2.ValidateVolumeCapabilitiesResponse
+
+    def get_create_object_method(self):
+        return self.servicer.ValidateVolumeCapabilities
+
+    @patch("controller.array_action.array_mediator_xiv.XIVArrayMediator._connect")
+    def setUp(self, connect):

```

Comment:
most of this function content is a duplication.
please move (/open a ticket to move) all common `setUp` functions content to a base class (e.g. `AbstractControllerTest`)

Reply:
done

Reply:
move [**all**](https://github.com/IBM/ibm-block-csi-driver/blob/386a02f1f5066b77571b531335055fdd2287eda1/controller/tests/controller_server/csi_controller_server_test.py#L1234) common `setUp` functions content

Reply:
done

Reply:
i still see some `setUp` content that can be unified under the base `setUp`, e.g. `self.request.volume_id`, but let's leave it for a future ticket